### ISSUE TITLE
Gloo rollout job failing when installing 1.12.x version of Enterprise

### ISSUE THREAD

**Issue by pseudonator** [solo-io/gloo#7047]
### Gloo Edge Version

1.12.x (latest stable)

### Kubernetes Version

1.22.x

### Describe the bug

Seeing the following error when doing a fresh install of `1.12.7` of GEE.

```
Error from server: error when creating "STDIN": admission webhook "gloo.gloo-system.svc" denied the request: resource incompatible with current Gloo snapshot: [Validating v1.Gateway failed: validating *v1.Gateway name:"gateway-proxy"  namespace:"gloo-system": failed to validate Proxy with Gloo validation server: HttpListener Error: ProcessingError. Reason: ratelimit server upstream not found name:"rate-limit"  namespace:"gloo-system"; HttpListener Error: ProcessingError. Reason: ratelimit server upstream not found name:"rate-limit"  namespace:"gloo-system"; HttpListener Error: ProcessingError. Reason: extauth server upstream not found name:"extauth"  namespace:"gloo-system"]
```

Digging into this further it seems like there is an issue with the execution of rollout job hooks.
The current order of hooks is such that Gloo rollout job gets executed before Gloo Edge Enterprise.

One this was temporarily fixed was to change `helm.sh/hook-weight`

### Steps to reproduce the bug

Deploy using Helm with the [Helm values](https://gist.github.com/pseudonator/7c3c9dfc41e0cd8523108cfc8255169e).

### Expected Behavior

Installation to complete with no errors

**Comment by ashsolo** [comment]
Closing as unable to reproduce.

**Comment by jenshu** [comment]
reopening, as customer is still running into this issue

**Comment by jenshu** [comment]
fix will be in GlooEE v1.11.44, v1.12.27, and v1.13.0-beta5
